### PR TITLE
feat: Support FALSE and TRUE strings in import

### PR DESCRIPTION
**Pull request by maikelarabori**
Adding support for uppercase TRUE and FALSE flags during the import validation.

### CODE REVIEW

File: dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java [hunk 1]
```diff
@@ -437,11 +437,11 @@ else if ( ValueType.INTEGER_ZERO_OR_POSITIVE == valueType && !MathUtils.isZeroOr
         {
             return "value_not_zero_or_positive_integer";
         }
-        else if ( ValueType.BOOLEAN == valueType && !MathUtils.isBool( value ) )
+        else if ( ValueType.BOOLEAN == valueType && !MathUtils.isBool( value.toLowerCase() ) )
         {
             return "value_not_bool";
         }
-        else if ( ValueType.TRUE_ONLY == valueType && !DataValue.TRUE.equals( value ) )
+        else if ( ValueType.TRUE_ONLY == valueType && !DataValue.TRUE.equals( value.toLowerCase() ) )
```
Comment (larshelge) [inline dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java]: Have you verified that we are not introducing a null pointer vulnerability here?

File: dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java [hunk 2]
```diff
@@ -437,11 +437,11 @@ else if ( ValueType.INTEGER_ZERO_OR_POSITIVE == valueType && !MathUtils.isZeroOr
         {
             return "value_not_zero_or_positive_integer";
         }
-        else if ( ValueType.BOOLEAN == valueType && !MathUtils.isBool( value ) )
+        else if ( ValueType.BOOLEAN == valueType && !MathUtils.isBool( value.toLowerCase() ) )
```
Comment (larshelge) [inline dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java]: Have you verified that we are not introducing a null pointer vulnerability here?

File: dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java (hunk 1 above)
Comment (maikelarabori) [reply dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java]: Hi @larshelge , actually at the beginning of this method we are already checking if the "value" is null.
But, I can make it safer for sure. I will fix it in a few minutes.

Thanks

Comment (maikelarabori) [reply dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java]: Fix added. Ready for another review.

File: dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java (hunk 2 above)
Comment (maikelarabori) [reply dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java]: Fixed too.

**Pull request merged**